### PR TITLE
Us12 comment profile

### PR DESCRIPTION
**Pull request by Benjamin-51**
Since the comment a profile page is supposed to be accessed from a user's profile page, but that page isn't made yet, you have to manually access it via the url /comment/{user_id}.
You have to be logged in in order to successfully send a comment, we made fixtures for the user entity, the password for each account is "aaaaaa".

### CODE REVIEW

**Review by ghost — DISMISSED** [review]
ok for me

**Review by ghost — DISMISSED** [review]
ok for me

**Review by RomainClair — CHANGES_REQUESTED** [review]
Migration files are not consistent with dev

**Review by Bruno-Fernandes-37 — DISMISSED** [review]

File: src/Controller/CommentController.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    }
+
+    /**
+     * If the form sent is valid, we send the user to this page to inform them it is valid.
+     * @Route("/valid", name="valid")
+     */
+    public function valid(Request $request): Response
+    {
+        return $this->render("comment/valid.html.twig");
+    }
```
Comment (RomainClair) [inline src/Controller/CommentController.php]: Why not just render comment/valid.html/twig in index() instead of redirecting ?

**Review by RomainClair — CHANGES_REQUESTED** [review]
You probably should add constraint validation in Comment to check user input in your form

**Review by CECDO — APPROVED** [review]
ok

**Review by RomainClair — APPROVED** [review]

**Pull request merged**